nucleus/__init__.py

```diff
@@ -1037,6 +1038,28 @@ def delete_slice(self, slice_id: str) -> dict:
         )
         return response
 
+    def delete_annotations(
+        self, dataset_id: str, reference_ids: list = None, keep_history=False
```

Comment:
Thanks for doing this... I said I would do it and then I got distracted by other things. You da best